extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -269,3 +270,79 @@
 .hidden {
 	display: none;
 }
+
+#WikiaArticle {
+	overflow: visible;
```

Comment:
I'm pretty sure this style will be loaded on every page—this might cause problems for some other pages then CNW. My first hacky idea that comes to my mind is to use JS to switch it when user clicks on language selection.